test/controllers/v1/permission_controller_test.exs

```diff
@@ -55,10 +55,11 @@ defmodule Cog.V1.PermissionControllerTest do
     conn = api_request(user, :post, "/v1/permissions",
                        body: %{"permission" => %{name: "test_perm"}})
     recv_perm = json_response(conn, 201)
-    assert recv_perm["permission"]["name"] == "test_perm"
-    id = recv_perm["permission"]["id"]
-    assert id != nil
-    assert Repo.get_by(Permission, id: id)
+    [perm|remaining] = recv_perm["permission"]["site"]
+    assert length(remaining) == 0
+    assert perm["name"] == "test_perm"
+    assert perm["id"] != nil
+    assert Repo.get_by(Permission, id: perm["id"])
```

Comment:
When getting something by the primary key you can use the shorter `Repo.get(Permission, perm["id"])`.